packages/augur-sdk/src/state/getter/ZeroXOrdersGetters.ts

```diff
@@ -62,11 +62,16 @@ export class ZeroXOrdersGetters {
     const outcome = params.outcome ? `0x0${params.outcome.toString()}` : undefined;
     const orderType = params.orderType ? `0x0${params.orderType}` : undefined;
 
-    let currentOrdersResponse = await db.ZeroXOrders.where('[market+outcome+orderType]').equals([
-      params.marketId,
-      outcome,
-      orderType
-    ]).toArray();
+    let currentOrdersResponse;
+    if (typeof outcome === 'undefined' && typeof orderType === 'undefined') {
+      currentOrdersResponse = await db.ZeroXOrders.toArray();
```

Comment:
shouldn't this still be querying based on market? As is this would return all existing orders on Augur.

Reply:
Also shouldn't this be an OR instead of an AND? I think there probably isnt a case where only one will be undefined but technically if there were this would raise an error

Reply:
Ah yeah. I did this fast to get it working but it definitely needs a fixup. There's a TODO at the beginning of this getter that makes me think a full refactor is needed.

I'll implement these suggestions real quick.

Reply:
Like this?
```
currentOrdersResponse = await db.ZeroXOrders.where('[market+outcome+orderType]').between(
        [params.marketId, Dexie.minKey, Dexie.minKey],
        [params.marketId, Dexie.maxKey, Dexie.maxKey],
      ).toArray();
```